apps/functions-subscription/api/internal.yaml

```diff
@@ -196,6 +196,30 @@ paths:
               schema:
                 $ref: '#/components/schemas/ProblemJson'
   /trials:
+    get:
+      operationId: listTrials
+      description: List all trials
+      summary: List all trials
+      parameters:
+        - $ref: '#/components/parameters/queryPageNumber'
+        - $ref: '#/components/parameters/queryPageSize'
```

Comment:
**issue(blocking)**: use cursor pagination

Reply:
Resolved in 728596289119be614fff33a07973a5f8e80d6b63